.circleci/config.yml

```diff
@@ -16,8 +16,52 @@ jobs:
       - run:
           name: "Sending coverage reports to CodeCov"
           command: "bash <(curl -s https://codecov.io/bash)"
+
+  build-docs:
+      docker:
+      - image: circleci/python:3.7.3
+      steps:
+      - checkout
+      - run:
+          name: "Installing Dependencies"
+          command: "sudo pip3 install -r requirements.txt"
+      - run:
+          name: "Setting environment variable"
+          command: "export PYTHONWARNINGS='error::UserWarning'"
+      - run:
+          name: "Building documenation using pdoc3"
+          command: "pdoc --html --skip-errors --output-dir docs ."
+      - persist_to_workspace:
+          root: ~/project/docs
+          paths: project
+
+  docs-deploy:
+      docker:
+      - image: node:8.10.0
+      steps:
+      - checkout
+      - add_ssh_keys:
+            fingerprints:
+                - "7f:f7:8d:79:e6:17:7a:4c:e3:ec:b7:05:55:57:fe:26"
+      - attach_workspace:
+            at: ~/project/docs/project
+      - run:
+            name: Install and configure dependencies
+            command: |
+                npm install -g --silent gh-pages@2.0.1
+                git config user.email "aditya.9395@gmail.com"
```

Comment:
Kindly do not add your own personal emails here.

Reply:
A valid git credential is needed to push the generated docs from the CircleCI containers to the gh-pages branch. Let me know how/what to replace it with.

https://circleci.com/docs/2.0/gh-bb-integration/#deployment-keys-and-user-keys

Reply:
Yes. Working on it.